src/plugins/wb-data-json/reports/pre-a11y-1.json

```diff
@@ -0,0 +1,589 @@
+{
+    "@context": "https://wet-boew.github.io/vocab/context/2023/assessment-report-en.json-ld",
+
+    "@type": ["earl:Assertion", "acr:AssessmentReport"],
+
+    "earl:subject": {
+        "@id": "_:subject",
+        "dct:references": "https://wet-boew.github.io/gcweb-compiled-demos/wetboew-demos/wb-data-json/data-json-en.html",
+        "@type": [
+            "earl:TestSubject",
+            "schema:WebPage"
+        ],
+        "dct:description": "Accessibility conformance reports of Data JSON",
+        "earl:pointer": {
+            "@type": "oa:CssSelector",
+            "@value": "data-json-replacewith,data-json-after,data-wb-json,data-json-append,data-json-before,data-json-prepend,data-json-replace,data-json-replacewith"
```

Comment:
Please adjust to make it a valid CSS selector which we can use as a parameter with the JS function `document.querySelectorAll( "" )` to find all evaluated HTML elements

Reply:
done